unicode_gen_def.h

```diff
@@ -288,4 +299,7 @@ DEF(XID_Start, "XIDS")
 /* internal tables with index */
 DEF(Cased1, "")
 
+/* unused by us */
+DEF(InCB, "")
```

Comment:
This is Indic_Conjunct_Break and I believe it's not supposed to be JS-visible because it isn't in V8 and test262 doesn't test it but I did see a reference to it in Servo's code base so... dunno :shrug:

It's not visible in qjs at any rate.